internals/processes.py

```diff
@@ -478,18 +482,67 @@ def process_to_dict(process):
   ]
 
 
+FEATURE_ROLLOUT_STAGE = ProcessStage(
```

Comment:
Please add a comment to FEATURE_ROLLOUT_STAGE to explain that this is a stage that can be inserted into the stages for non-enterprise features that are marked as breaking.